test/jdk/java/lang/invoke/indify/Indify.java

```diff
@@ -41,8 +43,8 @@
  * <p>
  * Static private methods named MH_x and MT_x (where x is arbitrary)
  * must be stereotyped generators of MethodHandle and MethodType
- * constants.  All calls to them are transformed to {@code CONSTANT_MethodHandle}
- * and {@code CONSTANT_MethodType} "ldc" instructions.
```

Comment:
Don't need to rename these, as they are from the JVMS 
https://docs.oracle.com/javase/specs/jvms/se22/html/jvms-4.html#jvms-4.4-210